draft-ietf-httpbis-header-structure.md

```diff
@@ -181,7 +181,7 @@ This section defines the abstract value types that can be composed into Structur
 
 ## Dictionaries {#dictionary}
 
-Dictionaries are ordered maps of key-value pairs, where the keys are short, textual strings and the values are items ({{item}}). There can be one or more members, and keys are required to be unique.
+Dictionaries are ordered maps of key-value pairs, where the keys are short, textual strings and the values are items ({{item}}) or lists thereof ({{list}}). There can be one or more members, and keys are required to be unique.
```

Comment:
Hmm, I don't think the reference to `list` is appropriate here; probably just say "arrays thereof."

Reply:
:+1: